packages/account/src/Sections/Security/ConnectedApps/connected-apps.tsx

```diff
@@ -43,7 +44,9 @@ const ConnectedApps = observer(() => {
                 {is_error && <ErrorComponent />}
                 {connected_apps.length ? (
                     <div className='connected-apps__content--wrapper'>
-                        <div>TODO: Replace this div with message component</div>
+                        <div>
+                            <ConnectedAppsInfo />
```

Comment:
will this work without `<div>` wrapper?
just `<ConnectedAppsInfo  />`

Reply:
Changed